doc/start/documenting-ceph.rst

```diff
@@ -60,32 +60,33 @@ instead of compiling program source. This sequence (the sequence of building the
 Get the Source
 --------------
 
-Ceph documentation lives in the Ceph repository right alongside the Ceph source
-code under the ``ceph/doc`` directory. For details on github and Ceph,
-see :ref:`Get Involved`.
+The source of the Ceph documentation is a colletion of ReStructured Text files
```

Comment:
Ceph documentation is a colleCtion of ReStructured Text (RST) files

Reply:
Good catch.

I wish we had the budget and manpower to have a bounty sytem for typos, the way Don Knuth does.